SDK.md

```diff
@@ -0,0 +1,173 @@
+# CloudEvents SDK
+
+The intent of this document to describe bare minimum set of requirements for a new SDKs.
+
+## Status of this document
+
+Since [CloudEvent spec](spec.md) still considered as a working draft this document also suppose to be considered as a working draft.
+
+## Contribution acceptance
+
+Being an open source community CloudEvents team is open for a new members as well open to their contribution.
+In order to ensure that an SDK is going to be supported and maintained CloudEvents community would like to ensured that:
+
+ - a person (developer, committer) is going to become a maintainer
+ - a person commits to support ongoing changes to the [CloudEvent spec](spec.md)
+
+## Officially maintained software development kits (SDK)
+
+Software Development Kit (SDK) is a set of helpers designed and implemented to enhance and speed up a CloudEvents integration.
+As part of community efforts CloudEvents team committed to support and maintain the following SDKs:
+
+ - [Go SDK](https://github.com/cloudevents/sdk-go)
+ - [Java SDK](https://github.com/cloudevents/sdk-java)
+ - [Python SDK](https://github.com/cloudevents/sdk-python)
+ - [CSharp](https://github.com/cloudevents/sdk-csharp)
+ - [JavaScript SDK](https://github.com/cloudevents/sdk-javascript)
+
+## SDK contribution
+
+That's said, being an open source community CloudEvents is all about building open for contribution.
+In order to keep the development/user experience the same (as much as possible) CloudEvents team agreed to establish minimum requirements for a new SDK.
+
+### Technical requirements
+
+Supports CloudEvents spec milestones and ongoing development version.
+HTTP transport support (both structured and binary).
+Widely used programming language version.
+
+
+### Preferable API signature guidelines
+
+In order to remain developer/user UX the same among existing officially supported SDKs CloudEvents team asks maintainers to align with the following API signatures.
+Please consider the following code as pseudo-code.
+
+#### Event object constructor API
+
+Event build considered to be an event constructor:
+```
+    v01.Event()
+```
+
+#### Event object setters API 
+
+This particular code sample represents bare minimum number of setters:
+```
+    v01.Event().
+    WithContentType("application/json").
```

Comment:
Minor thing, but to avoid people getting confused since you use the term "setters", it might be better to use `SetXXX` instead of `WithXXX`.